src/platform-includes/sourcemaps/upload/primer/javascript.angular.mdx

```diff
@@ -0,0 +1,10 @@
+To upload your Angular project's source maps to Sentry, we recommend one of these two options:
+
+- [**Angular CLI and Sentry Webpack Plugin**](./angular-webpack/) <br/>
+  Use the Angular CLI, a custom Angular builder and the Sentry Webpack plugin to set releases and upload source maps automatically when running `ng build`.
+- [**Sentry CLI**](./cli/)<br/>
+  Upload source maps manually using Sentry-CLI.
+
+These options work well with Angular projects out of the box. For other bundlers or more advanced projects and configurations, take a look at the following guides and options for uploading sourcemaps:
+
+<PageGrid exclude={['angular-webpack', 'cli']}/>
```

Comment:
If we updated the content per my comment above, we could delete this whole section.